massa-node/base_config/config.toml

```diff
@@ -132,6 +132,8 @@
     max_operations_propagation_time = 32000
     # time threshold after which endorsement are not propagated
     max_endorsements_propagation_time = 48000
+    # operations sender(channel) capacity
+    ws_operations_capacity = 128
```

Comment:
That's a really low number as we have multiple thousands of op in 1 second and you can receive batch that are way more huge then 128. I think you can put 5_000 at least

Reply:
Yes, I knew that you'll give us a good one

Reply:
done